roles/nsx_node/tasks/main.yml

```diff
@@ -0,0 +1,116 @@
+---
+- name: NSX | Download CNI rpm
```

Comment:
It would probably be best to have all RPMs downloaded in one task using `with_items` (same for install)

It would be convenient to have a repo with these RPMs instead of downloading and installing them one by one.

Reply:
In a couple of months we plan to have the nsx-node-agent daemonset bootstraping the nodes with the cni and ovs. This will make the whole nsx_node role redundant. I will send a separate pr removing the role once we are ready. So I wouldn't spend much time for it. Anyway, I will go and fix what you suggested!

Reply:
I see, in that case not using a repo is fine for now. However grouping tasks would be a good idea and simplify debugging (e.g. one package failed to download when others have already been installed)